frontend/apps/reader/modules/readergesture.lua

```diff
@@ -259,7 +259,7 @@ function ReaderGesture:init()
             end
         end
         if reset then
-            logger.info("UI language direction changed: resetting some gestures to direction default")
+            logger.info("UI language direction changed: Resetting some gestures to default direction.")
```

Comment:
```suggestion
            logger.info("UI language direction changed: Resetting some gestures to directional defaults.")
```

Reply:
I don't like this one either :) even if directional is the english adjective for direction, I'd rather keep the word direction to again be explicite (as we're not all native english speakers, simple english is sometimes better in comments).
But it's just a log, and if you'd really want it to be clear and unambiguous, you'd use:
```diff
-UI language direction changed: resetting some gestures to direction default
+UI language direction changed: Resetting some gestures to their default for the new direction.
```

Reply:
The original sentence would be better as "direction defaults."

As an aside, for comments & logs I purposefully try not suggest corrections or improvements as long as they're comprehensible and not ungrammatical. Because there the goal is the future maintainability of the code.

Reply:
Using the comment suggest functionality to fix this up for merge.
```suggestion
            logger.info("UI language direction changed: resetting some gestures to their default for the new direction.")
```

Reply:
@comradekingu You didn't enable push access. Could you please fix up/revert the random comments so the PR can be merged?

Reply:
@Frenzie I did, but your are confusing the plural "gestures" with the singular "default". I didn't randomly change that.

Reply:
@comradekingu 
![Screenshot_2020-02-12_18-36-25](https://user-images.githubusercontent.com/202757/74361114-982a6500-4dc6-11ea-9664-6468853fb6bc.png)

> but you are confusing the plural "gestures" with the singular "default".

You can certainly say English generally prefers a plural noun for a repeated idea, but that's besides the point.

Reply:
Except it's "value" taking the plural in his suggestion. default values OR defaults. defaults values would be wrong, though.

Reply:
@NiLuJe I just edited it to "default values"; it just said "default" previously because I copied it straight from @poire-z as a statement. ;-)

I figured most people would miss the subtle statement; I wasn't quite expecting it to be caught yet still missed. ^_^

Reply:
@Frenzie: Ah, okay, now it makes sense ;). Thanks.